http_api/Cargo.toml

```diff
@@ -27,7 +27,8 @@ helper_functions = { workspace = true }
 http_api_utils = { workspace = true }
 itertools = { workspace = true }
 liveness_tracker = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order